x-pack/plugins/security_solution/public/common/components/query_bar/index.test.tsx

```diff
@@ -82,130 +85,7 @@ describe('QueryBar ', () => {
-              searchable: true,
-              type: 'string',
-            },
-            {
-              aggregatable: true,
-              name: 'agent.ephemeral_id',
-              searchable: true,
-              type: 'string',
-            },
-            {
-              aggregatable: true,
-              name: 'agent.hostname',
-              searchable: true,
-              type: 'string',
-            },
-            {
-              aggregatable: true,
-              name: 'agent.id',
-              searchable: true,
-              type: 'string',
-            },
-            {
-              aggregatable: true,
-              name: 'agent.test1',
-              searchable: true,
-              type: 'string',
-            },
-            {
-              aggregatable: true,
-              name: 'agent.test2',
-              searchable: true,
-              type: 'string',
-            },
-            {
-              aggregatable: true,
-              name: 'agent.test3',
-              searchable: true,
-              type: 'string',
-            },
-            {
-              aggregatable: true,
-              name: 'agent.test4',
-              searchable: true,
-              type: 'string',
-            },
-            {
-              aggregatable: true,
-              name: 'agent.test5',
-              searchable: true,
-              type: 'string',
-            },
-            {
-              aggregatable: true,
-              name: 'agent.test6',
-              searchable: true,
-              type: 'string',
-            },
-            {
-              aggregatable: true,
-              name: 'agent.test7',
-              searchable: true,
-              type: 'string',
-            },
-            {
-              aggregatable: true,
-              name: 'agent.test8',
-              searchable: true,
-              type: 'string',
-            },
-            {
-              aggregatable: true,
-              name: 'host.name',
-              searchable: true,
-              type: 'string',
-            },
-            {
-              aggregatable: false,
-              name: 'nestedField.firstAttributes',
-              searchable: true,
-              type: 'string',
-            },
-            {
-              aggregatable: false,
-              name: 'nestedField.secondAttributes',
-              searchable: true,
-              type: 'string',
-            },
-          ],
-          title: 'filebeat-*,auditbeat-*,packetbeat-*',
-        },
-      ],
-      isLoading: false,
-      isRefreshPaused: true,
-      query: {
-        language: 'kuery',
-        query: 'here: query',
-      },
-      refreshInterval: undefined,
-      savedQuery: undefined,
-      showAutoRefreshOnly: false,
-      showDatePicker: false,
-      showFilterBar: true,
-      showQueryInput: true,
-      showSaveQuery: true,
-      showSubmitButton: false,
-    });
+    expect(searchBarProps).toMatchSnapshot();
```

Comment:
Can we avoid using snapshot tests? They often cause more maintenance issues than the value they provide due to their brittleness and lack of context (i.e., it's difficult to identify what matters in a test and what's not).

In this specific case, the snapshot verifies the shape of the component props (if I get it right), which is something that should already be covered by appropriate TypeScript types. So we can just remove it.